media-gfx/freecad/freecad-0.19.2-r4.ebuild

```diff
@@ -79,7 +78,7 @@ RDEPEND="
 		dev-libs/openssl:=
 		net-misc/curl
 	)
-	fem? ( sci-libs/vtk:=[boost,python,qt5,rendering,${PYTHON_SINGLE_USEDEP}] )
+	fem? ( sci-libs/vtk:=[boost(+),python,qt5,rendering,${PYTHON_SINGLE_USEDEP}] )
```

Comment:
Requires a revbump usually, but we still have the flag and I guess you're just preparing for possible removal a while in the future? (not with the next version?)

Reply:
I can do the revbump, of course. Yes this is in preparation of a possible removal of the flag from vtk-9.1.0+.
I can do this in a separate commit later on, to avoid a revbump for now.